core/pom.xml

```diff
@@ -183,7 +183,24 @@
             <version>1.9.20</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-openfeign</artifactId>
+            <version>4.1.3</version>
+        </dependency>
+
     </dependencies>
+    <dependencyManagement>
```

Comment:
What is the reason for using `dependencyManagement` only for this dependency?

Reply:
My bad, i'll refactored this!